src/Liquidator.sol

```diff
@@ -96,8 +96,8 @@ contract Liquidator is Owned {
         factory = factory_;
         initiatorRewardWeight = 1;
         penaltyWeight = 5;
-        startPriceMultiplier = 110;
-        minPriceMultiplier = 50;
+        startPriceMultiplier = 150;
+        minPriceMultiplier = 60;
         cutoffTime = 14_400; //4 hours
         base = 1e18;
```

Comment:
Can't set this to 1e18! should be smaller than 1

Reply:
set to 999807477651317446 = 3600s halflife, 14_400 cutoff